cypress/integration/MultiSelect/position/index.js

```diff
@@ -22,6 +22,21 @@ Given(
     }
 )
 
+Given('the input is empty', () => {
+    cy.get('[data-test="dhis2-uicore-select-input"]')
+        .should('exist')
+        .should('have.length', 1)
+        .then(inputs => {
+            const $input = inputs[0]
+            const inputRect = $input.getBoundingClientRect()
+            const height = inputRect.height
+
+            cy.wrap(height).as('emptyInputHeight')
```

Comment:
you could just return the height here and then chain the `.as` command on the outer chain

Reply:
Ah yeah that's better. Fixed in 087b635